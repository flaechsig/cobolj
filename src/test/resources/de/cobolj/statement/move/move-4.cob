IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO-WORLD.

DATA DIVISION.
  WORKING-STORAGE SECTION.
  01 WS-GROUP1.
     02 WS-ITEM1 PIC X(2) VALUE 'AB'.
     02 WS-ITEM2 PIC X(2) VALUE 'CD'.
  01 WS-GROUP2.
     02 WS-ITEM1 PIC X(2) VALUE '12'.
     02 WS-ITEM1 PIC X(2).
     
PROCEDURE DIVISION.
DISPLAY WS-GROUP1 " " WS-GROUP2.
MOVE WS-GROUP1 TO WS-GROUP2.
DISPLAY WS-GROUP1 " " WS-GROUP2.
STOP RUN.