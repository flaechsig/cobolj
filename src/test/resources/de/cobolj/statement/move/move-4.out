ABCD 12  
ABCD ABCD
